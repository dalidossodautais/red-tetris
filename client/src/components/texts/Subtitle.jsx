import styled from "styled-components";

export default styled.span.attrs((props) => ({
  color: props.theme.primaryTextColor,
}))`
  color: ${(props) => props.color};
  text-align: ${(props) => props.textAlign};
  text-overflow: ellipsis;
`;
