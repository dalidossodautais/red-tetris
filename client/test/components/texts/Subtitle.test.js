import React from "react";
import renderer from "react-test-renderer";
import Subtitle from "../../../src/components/texts/Subtitle";

describe("Subtitle", () => {
  test("Standard", () => {
    const component = renderer.create(<Subtitle />);
    // rendering
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    // re-rendering
    tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
