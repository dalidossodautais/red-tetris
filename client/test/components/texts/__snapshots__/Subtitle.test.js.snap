// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subtitle Standard 1`] = `
<span
  className="sc-bdfBwQ jVxTIo"
/>
`;

exports[`Subtitle Standard 2`] = `
<span
  className="sc-bdfBwQ jVxTIo"
/>
`;
